src/PKSim.Presentation/Presenters/ConfigurableLayoutPresenter.cs

```diff
@@ -40,6 +40,7 @@ public void AddViews(IEnumerable<IView> views)
          if (allViews.Count > 1)
          {
             var configurableView = _configurableContainerLayoutViewFactory.Create();
+            configurableView.StartAddingViews();
```

Comment:
@Yuri05 I have added a step here but this is in fact not required as the views are only redrawn when visible. At any rate, it does not hurt and is symmetrical